### PR TITLE
WRR-7946: Update .travis.yml to use dist jammy

### PR DESCRIPTION
**Pull request by daniel-stoian-lgp**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Travis was failing  sue to wrong dist set in .travis.yml

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Update .travis.yml to use dist jammy

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRR-7946

### Comments
Enact-DCO-1.0-Signed-off-by: Daniel Stoian (daniel.stoian@lgepartner.com)

### CODE REVIEW

**Review by adrian-cocoara-lgp — APPROVED** [review]

**Comment by hong6316** [comment]
I think it's `dist` issue.
Please try `dist` from `focal` to `jammy`

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 dist: focal
```
Comment (hong6316) [inline .travis.yml]:
```suggestion
dist: jammy
```

File: .travis.yml [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
 language: node_js
 node_js:
-    - node
+    - "22"
```
Comment (hong6316) [inline .travis.yml]:
```suggestion
    - "node"
```

**Review by hong6316 — COMMENTED** [review]
I think it's `dist` issue.
Please try `dist` from `focal` to `jammy`, and revert `node_js`

File: .travis.yml [hunk 3]
```diff
@@ -1,7 +1,7 @@
-dist: focal
+dist: jammy
 language: node_js
 node_js:
-    - node
+    - "node"
```
Comment (daniel-stoian-lgp) [inline .travis.yml]:
```suggestion
    - node
```

**Review by hong6316 — APPROVED** [review]
👍

**Pull request merged**